modules/ossm-rn-fixed-issues.adoc

```diff
@@ -41,3 +41,16 @@ To check HTTPS or TCP traffic by using an ingress gateway, you must have an exis
 * link:https://issues.jboss.org/browse/OSSM-93[OSSM-93] IstioConfigList can't filter by two or more names.
 +
 * link:https://issues.jboss.org/browse/OSSM-99[OSSM-99] Workloads generated from direct Pod without labels may crash Kiali.
+
+[id="ossm-rn-fixed-issues-maistra_{context}"]
+== Maistra fixed issues
+
+* link:https://issues.jboss.org/browse/MAISTRA-932[MAISTRA-932] Added the *requires* metadata to add dependency relationship between Jaeger operator and Elasticsearch operator. Ensures that when the Jaeger operator is installed, it automatically deploys the Elasticsearch operator if it is not available.
+
+* link:https://issues.jboss.org/browse/MAISTRA-933[MAISTRA-933] Upgraded to Istio version 1.1.17.
+
+* link:https://issues.jboss.org/browse/MAISTRA-1001[MAISTRA-1001] Fixed issue in which a serverless operator on AWS caught a segmentation fault and failed.
+
+* link:https://issues.jboss.org/browse/MAISTRA-833[MAISTRA-833] Knative service starts to return a 503 status code when the cluster runs for a long period of time. 
```

Comment:
"Fixed an issue where pilot would stop delivering configuration after a lot of namespace deletions and re-creations"